pkg/util/goroutinemap/goroutinemap.go

```diff
@@ -58,6 +58,10 @@ type GoRoutineMap interface {
 	// necessary during tests - the test should wait until all operations finish
 	// and evaluate results after that.
 	Wait()
+
+	// IsOperationPending returns true if the operation is pending, otherwise
+	// returns false
+	IsOperationPending(operationName string) bool
```

Comment:
Would be nice to have unit tests for new method. You can add them in a separate PR if you'd like.